content/blogs/newsletter-100.md

```diff
@@ -0,0 +1,64 @@
+---
+title:       "Newsletter 100 - January 2021 news"
+author:      "Extravert34"
+date:        2021-02-06
+tags:        [ newsletter ]
+---
+
+Hello ReactOS followers!
+This is an attempt to restore the tradition of annual newsletters covering our resent development efforts.
+The previous [Newsletter 99](/blogs/newsletter-99) was in 2013!
+
+This month we were mostly focused on the kernel work, merging pull requests and fixing regressions
+
+## Memory manager and Common cache work
+
+Being in development for almost half a year, this work by Jérôme Gardou finally landed in the master branch.
+This change refactors many aspects of Section Objects feature in NT-compatible kernel, making it more compatible with Windows.
+
+Here are notable changes from the user perspective:
+
+* A long-standing bug preventing to start executables from remote locations is fixed ([CORE-10038](https://jira.reactos.org/browse/CORE-10038)).
+* Open source Microsoft FAT filesystem driver is enabled by default now.
+* Better compatibility with FS drivers in general, `ntfs.sys` and `exfat.sys` drivers from Windows are now close to working state in ReactOS.
+* External drives with FAT filesystem can now be properly ejected
+
+## Changes for GCC amd64 build
+
+At the moment, we support building for amd64 target only with MSVC compiler.
```

Comment:
Suggestsion: At the moment, we support building for an amd64 target only with the MSVC compiler.